AI/VCAI/VCAI.h

```diff
@@ -138,7 +141,10 @@ class VCAI : public CAdventureAI
 
 	friend class FuzzyHelper;
 
+	std::map<TeleportChannelID, shared_ptr<TeleportChannel> > knownTeleportChannels;
 	std::map<const CGObjectInstance *, const CGObjectInstance *> knownSubterraneanGates;
+	ObjectInstanceID destinationTeleport;

```

Comment:
Oops, I think the way that after load it's will be set to -1, but as I get constructor probably only executed once. Yeah?

I didn't add it to serialization as it's only set to anything except -1 right before teleportation should occur and then teleportation AI should always occur before end of turn. Or valid game save is possible during AI turn?